bluesky/utils.py

```diff
@@ -1516,25 +1516,39 @@ def _rearrange_into_parallel_dicts(readings):
     return data, timestamps
 
 
+def is_movable(obj):
```

Comment:
I think it's better to include a docstring as for `Movable` class.

Reply:
thanks for catching that @dmgav